library/src/test/kotlin/io/github/wax911/library/annotation/processor/fragment/RegexFragmentAnalyzerTest.kt

```diff
@@ -3,7 +3,7 @@ package io.github.wax911.library.annotation.processor.fragment
 import org.junit.Assert.assertEquals
 import org.junit.Test
 
-class RegexFragmentAnalyzerSpec {
+class RegexFragmentAnalyzerTest {
```

Comment:
I had some mistakes in this test from my last PR, that Lucas found (👍). These were fixed.